pages/classes/mighty.md

```diff
@@ -42,7 +42,45 @@ On your turn, you can regain [Health](pages/combat/health.md) equal to 1d10 + yo
 
 </header>
 
-Choose an archetype and gain it's features. The Mighty archetypes include: [Defender](pages/classes/mighty.md?id=defender).
+Choose an archetype and gain it's features. The Mighty archetypes include: [Berserker](pages/classes/mighty.md?id=berserker) and [Defender](pages/classes/mighty.md?id=defender).
+
+<header>
+
+## Berserker
+
+<p class="subheading">Mighty archetype</p>
+
+</header>
+
+You fight with blind fury, without regard for your own safety.
+
+| Level | Archetype Features |
+| ----  | ------------------ |
+| 1st   | Savagery, Surge: Rage, Proficient: Intimidation |
+
+<header>
+
+### Savagery
+
+<p class="subheading">1st-level Berserker archetype feature</p>
+
+</header>
+
+When you roll a 1 or 2 on a damage die for an attack you make with a melee weapon, you can reroll the die.
+
+<header>
+
+### Surge: Rage
+
+<p class="subheading">1st-level Berserker archetype feature</p>
+
+</header>
+
+On your turn, you can use your surge to enter a rage as a bonus action. While raging, you gain <strong>1d10 + your Level</strong> [Temporary Hit Points]().
```

Comment:
This now requires the rule for temporary hit points. It's an added complication, but does make this class feature more streamlined and balanced.